mmv1/third_party/terraform/resources/resource_compute_instance_template.go.erb

```diff
@@ -281,7 +281,27 @@ func resourceComputeInstanceTemplate() *schema.Resource {
 				Computed:    true,
 				Description: `The unique fingerprint of the metadata.`,
 			},
-
+<% unless version == 'ga' -%>
+			"network_performance_config": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				ForceNew:    true,
+				Computed:    true,
+				Description: `Configures network performance settings for the instance. If not specified, the instance will be created with its default network performance configuration.`,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"total_egress_bandwidth_tier": {
+							Type:         schema.TypeString,
+							Optional:     true,
```

Comment:
See my previous comment about switching to `Required: true`

Reply:
Bleh - you were explicit here and I missed it. 🙏